acceptance/lib/facter/acceptance/base_fact_utils.rb

```diff
@@ -79,9 +79,15 @@ def debian_expected_facts(agent)
         if agent['platform'] =~ /amd64/
           os_arch     = 'amd64'
           os_hardware = 'x86_64'
+          processor_model_pattern = /(Intel\(R\).*)|(AMD.*)/
+        elsif agent['platform'] =~ /aarch64/
+          os_arch     = 'aarch64'
+          os_hardware = 'aarch64'
+          processor_model_pattern = // # facter doesn't figure out the processor type on these machines
```

Comment:
Would you be able to open a ticket about the processor.models fact missing? It looks like the AWS Graviton processors don't list their model information in `/proc/cpuinfo`, where Facter typically looks for model information, but we can pull it from other sources (like `lscpu`).

Reply:
Thanks @mhashizume, 
I have created the [ticket](https://perforce.atlassian.net/browse/FACT-3439) and mentioned it in the comment.